### PR TITLE
Initial layout implementation

### PR DESCRIPTION
**Pull request by limavfabio**
This PR implements the initial layout of the Calculator component for the Math Magicians Project

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,11 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <main>
+      <Calculator />
+    </main>
```
Comment (phelian23) [inline src/App.js]: - [x] As requested in the project requirements, at this stage, the App component should be a class-based component. Kindly implement this.

File: src/styles/Calculator.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.calculator-body {
+  display: grid;
+  grid-template: repeat(6, 1fr) / repeat(4, 1fr);
+}
+
```
Comment (phelian23) [inline src/styles/Calculator.css]: - [x] Kindly reduce the width of the calculator to match the width of the given design
Your design
Capture
Given template
figma

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @limavfabio,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Following GitHub Flow
✔️ Following requirements
✔️ Descriptive PR Title
✔️ Descriptive PR Summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @limavfabio,

Good job on making the changes asked by the previous reviewer👍

Your project is complete!  

GenieYouAreDoingGreatGIF (2)

## Highlights:

✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ README is okay
✅ App is created with class-based components
✅ Your design looks great 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**